src/libpgmoneta/configuration.c

```diff
@@ -1006,17 +1011,39 @@ pgmoneta_read_configuration(void* shm, char* filename)
                {
                   if (!strcmp(section, "pgmoneta"))
                   {
-                     if (as_int(value, &config->retention))
+                     int retention_len = 0;
+                     int retention[MAX_RETENTION_LENGTH];
```

Comment:
Keep it as variables - not as an array

Reply:
Well in config struct thery are kept as variables, this is just for easy parsing. If we don't do this we will need to do the same thing four times, there will be a lot code duplication.